src/Modal.php

```diff
@@ -56,21 +56,25 @@ final class Modal extends Widget
     private ?string $footer = null;
     private array $footerOptions = [];
     private ?string $size = self::SIZE_DEFAULT;
-    private ?array $closeButton = ['class' => 'btn-close'];
-    private ?array $toggleButton = [];
     private array $options = [];
     private bool $encodeTags = false;
     private bool $fade = true;
     private bool $staticBackdrop = false;
     private bool $scrollable = false;
     private bool $centered = false;
     private ?string $fullscreen = null;
+    protected string|Stringable $toggleLabel = 'Show';
 
     public function getId(?string $suffix = '-modal'): ?string
     {
         return $this->options['id'] ?? parent::getId($suffix);
     }
 
+    public function bsToggle(): string
```

Comment:
Is it possible to give it a more clear name?

Reply:
`toggleType` maybe?

Reply:
> `toggleType` maybe?

`toggleComponent`?

Reply:
Yeah.

Reply:
> Yeah.

Done